src/plugins/intel_cpu/src/utils/shape_inference/memory_accessor.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include <vector>
+#include "cpu_memory.h"
+#include "openvino/core/shape.hpp"
+#include "tensor_data_accessor.hpp"
+#include <ie_ngraph_utils.hpp>
```

Comment:
Is it required?
Maybe there is some equivalent in "openvino" includes?

Reply:
fixed it